src/SailthruChannel.php

```diff
@@ -67,6 +68,23 @@ public function send(
                 static::getDefaultVars()
             );
 
+            if (config('services.sailthru.whitelist.enabled') === true) {
```

Comment:
Does this mean that when sailthru whitelisting is enabled, we will only route emails specific whitelisted domains?